### PR TITLE
Changes to add project API support for FC host WWPNs.

### PR DESCRIPTION
**Pull request by chitranm**
Here is a user story for this change - https://rally1.rallydev.com/#/693886629169d/iterationstatus?detail=%2Fuserstory%2F733027848789

### CODE REVIEW

**Review by harinathakotha — APPROVED** [review]

File: v1/api/swagger/components/schemas/Host.yaml
Comment (mchuang3) [inline v1/api/swagger/components/schemas/Host.yaml:197]: WWPN should be "world wide port names" -
```suggestion
        description: FC host port world wide port names
```

File: v1/api/swagger/components/schemas/NewHost.yaml
Comment (mchuang3) [inline v1/api/swagger/components/schemas/NewHost.yaml:154]:
```suggestion
    description: FC host port world wide port names
```

File: v1/pkg/client/model_host.go
Comment (mchuang3) [inline v1/pkg/client/model_host.go:81]: Same for the other files below.
```suggestion
	// FC host port world wide port names
```

File: v1/pkg/client/api/openapi.yaml
Comment (mchuang3) [inline v1/pkg/client/api/openapi.yaml:2898]: This doesn't look right. Each entry is one WWPN, not WWPNs. Not sure how this got generated this way. Perhaps add real WWPN examples in the yaml files.
```suggestion
        WWPNs:
        - WWPN
        - WWPN
```

**Review by mchuang3 — CHANGES_REQUESTED** [review]

File: v1/pkg/client/api/openapi.yaml
Comment (chitranm) [reply v1/pkg/client/api/openapi.yaml:2898]: Hi @mchuang3, In the model/host.go, it is defined as array of string **WWPNs []string**. So I added it here in the same way. And I assume that WWPNs are the read only member and not with writeable.

Comment (mchuang3) [reply v1/pkg/client/api/openapi.yaml:2898]: Yes, it's read-only. However, "WWPN" or "WWPNs" are not good examples. Please use a real example of WWPN, similar to UUID examples in the other Host attributes.

File: v1/pkg/client/model_host.go
Comment (chitranm) [reply v1/pkg/client/model_host.go]: Hi @mchuang3, this is fixed as part of auto generation of code after making changes in host.yaml file.

File: v1/api/swagger/components/schemas/NewHost.yaml
Comment (shi98382) [inline v1/api/swagger/components/schemas/NewHost.yaml:159]: If they're mocked, then replace with real WWPNs as example.

File: v1/api/swagger/components/schemas/Host.yaml
Comment (shi98382) [inline v1/api/swagger/components/schemas/Host.yaml:197]: It should be FC HBA world wide port names. Make global change for all

**Comment by shi98382** [comment]
@mchuang3 this change might need to be further confirmed with Dilip in term of priority. Agree to block for now.

File: v1/api/swagger/components/schemas/NewHost.yaml
Comment (chitranm) [reply v1/api/swagger/components/schemas/NewHost.yaml:159]: I have taken the real example from an email sent by Prashanth for existing WWPNs.  
20:31:01:02:ad:f2:a8:e5
21:31:01:02:ad:f2:a8:e5
I have just replaced 20 and 21 with 18 and 19 for capturing here as an example.

File: v1/api/swagger/components/schemas/Host.yaml
Comment (chitranm) [reply v1/api/swagger/components/schemas/Host.yaml:197]: Ok.

**Comment by chitranm** [comment]
> @mchuang3 this change might need to be further confirmed with Dilip in term of priority. Agree to block for now.

@shi98382 and @mchuang3, does **"Agree to block for now"**  means that I don't have to work on it now?

File: v1/api/swagger/components/schemas/NewHost.yaml
Comment (shi98382) [reply v1/api/swagger/components/schemas/NewHost.yaml:159]: The first octet contains vendor information that can not be always replaced with a mocked value.

**Comment by mchuang3** [comment]
> > @mchuang3 this change might need to be further confirmed with Dilip in term of priority. Agree to block for now.
> 
> @shi98382 and @mchuang3, does **"Agree to block for now"** means that I don't have to work on it now?

@chitranm no, it just means we're blocking until the comments are addressed. Please update PR according to Harris's comments, regarding wordings & sample WWPN values. Thanks.

File: v1/api/swagger/components/schemas/NewHost.yaml
Comment (chitranm) [reply v1/api/swagger/components/schemas/NewHost.yaml]: Done.

**Review by mchuang3 — COMMENTED** [review]
@shi98382 looks like all your comments have been addressed. Please review again and approve if you have no other issues. Once you approve, I'll remove my block. Thanks.

**Review by shi98382 — APPROVED** [review]

**Review by mchuang3 — APPROVED** [review]

File: v1/api/swagger/components/schemas/NewHost.yaml
Comment (mchuang3) [inline v1/api/swagger/components/schemas/NewHost.yaml:159]: @chitranm This should be removed from here since this is for creating new Hosts. WWPNs are h/w info that the user should not provide. Sorry I missed this previously.

Comment (chitranm) [reply v1/api/swagger/components/schemas/NewHost.yaml:159]: Thanks @mchuang3. So I should remove it only from NewHost.yaml right? And keep the changes in Host.yaml as is?

**Pull request merged**
